userdocs/profiles.dev/docs/author-docs/nested-profiles.md

```diff
@@ -0,0 +1,33 @@
+---
+sidebar_position: 5
+---
+
+# Nested profile artifacts
+
+Profiles can refer to other profiles in their list of artifacts.
+
+Simply use the `source` field:
```

Comment:
Technically, using `profile:` is the one you are using combined with source. Not just `source`.

Reply:
yeh i forgot about that higher level key